ui/brave_ads/message_popup_view_win.cc

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/ui/brave_ads/message_popup_view.h"
+
+#include <windows.h>
+
+#include "ui/display/win/screen_win.h"
+
+namespace gfx {
```

Comment:
Usually, we don't use forward-declaration in cc file.
It seems `rect.h` is included indirectly. Just deleting this or including it would be better.

Reply:
Fixed, was a copy/paste from other code :-(